CHANGELOG.md

```diff
@@ -10,6 +10,15 @@ internal API changes are not present.
 Main (unreleased)
 -----------------
 
+### Bugfixes
+
+- Fix a bug where custom components would not shadow the stdlib. (@wildum)
```

Comment:
Since we haven't release 1.1.0-rc.0 yet (due to issues), can you put this into the 1.1.0-rc.0 section and I'll cherry-pick it over?